scipy/signal/ltisys.py

```diff
@@ -1086,6 +1096,10 @@ def lsim2(system, U=None, T=None, X0=None, **kwargs):
     given to `lsim2` are passed on to `odeint`.  See the documentation
     for `scipy.integrate.odeint` for the full list of arguments.
 
+    If (num, den) is passed in for `system`, coefficients for both the
```

Comment:
I actually was trying to match it to the argument signatures and current documentation, some of which use ```*system``` and others which use ```system```.  Should I just use ```system``` for all of them?